### PR TITLE
Downgrades: Make text clear that downgrades occur immediately

### PR DESCRIPTION
**Pull request by aaronrobertshaw**
#### Changes proposed in this Pull Request

Make it clearer to users that downgrading a subscription will come into effect immediately. Relates to #37982

This will add a downgrade warning message to:

* First step of the cancellation process
* Downgrade selection step

Example of original text:

> When you cancel your subscription, you'll be able to use WordPress.com Premium until your subscription expires. Once it expires, it will be automatically removed from your site.

##### Prerequisites:

* Must be a simple site (not Atomic)
* Needs to have at least Premium plan upgrade
* Auto-renewal must be enabled

##### Steps to view text changes

1. Visit URL: https://wordpress.com/me/purchases
2. Select the Business or Premium plan to be downgraded.
3. Choose "Cancel Subscription".
4. Check the cancellation information includes downgrade warning.
5. Click the "Cancel Subscription" button.
6. Choose "The plan was too expensive" for the cancellation reason.
7. Below that, select "I'm staying here and using the free plan" for your next adventure.
8. Click the "Next Step" button.
9. Confirm the displayed information contains downgrade warning.

##### Before / After Screenshots

Cancellation Step:

OriginalCancellationText

UpdatedCancellationText

Downgrade Step:

OriginalDowngradeText
UpdatedDowngradeText

### CODE REVIEW

File: client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ export class DowngradeStep extends Component {
 		const canRefund = !! parseFloat( refundAmount );
 		const amount = currencySymbol + ( canRefund ? refundAmount : planCost );
 		const isEnglishLocale = [ 'en', 'en-gb' ].indexOf( userUtils.getLocaleSlug() ) >= 0;
+		const downgradeWarning =
```
Comment (glendaviesnz) [inline client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx:41]: Sorry, I forgot to mention to you about translation - all new strings need to be translatable - see the other uses of translate() in this file for examples, and https://github.com/Automattic/i18n-calypso/blob/master/README.md give more details about translation options.

**Comment by aaronrobertshaw** [comment]
Hi @xpurichan, can you take a quick look at the new wording here for warning that downgrades occur immediately and make sure you are happy with it?

**Comment by DavidRothstein** [comment]
Looks pretty reasonable, but I don't think the text should be added to the cancellation screen at all.  Most users who cancel don't get the option to downgrade, right?  So this would just be confusing for them.  And it isn't really necessary, since for anyone who actually cancels (rather than changing their mind later and downgrading) the text seems correct as is.

I added the Marketing Team as a reviewer since the downgrade functionality was really built by them.  Not sure exactly who is best to review the wording right now though.

**Comment by glendaviesnz** [comment]
> but I don't think the text should be added to the cancellation screen at all

Thanks for the feedback David, we can easily remove it from the first cancel screen. We will wait and see if there is any comment from Marketing on the wording before we change it.

**Comment by aaronrobertshaw** [comment]
@Automattic/martech We've merged in these changes, let us know if you would like any changes.

**Comment by xpurichan** [comment]
@aaronrobertshaw sorry for the late reply! My team's virtual meetup happened last week, I was AFK for a day, and now I'm sprinting to catch back up. Ha! 

I think the wording is great. Regarding the cancellation step, I do actually think it's important to specify the difference between the cancellation vs downgrade process (one happens later and the other happens immediately). I think it's important because this wording is read by the user right away regardless of whether they're going to end up cancelling or downgrading in the later prompts. It was the incorrect wording about this which led to the GH report being written in the first place (though of course in that previous report we didn't have better wording later on in the process!)

I can understand what @glendaviesnz means about it being in an odd place; because the user hasn't yet decided to cancel or downgrade at this point in the process.

Honestly though, lots of users aren't reading every one of our messages word-by-word. My fear is them seeing the first message about the cancellation happening _later_ and continuing to expect that when the downgrade (even if the messaging on this next prompt says otherwise). Just my thoughts!

**Comment by DavidRothstein** [comment]
I still don't think it would be good to put this on the cancellation screen -- not only has the user not made a decision to cancel or downgrade yet, but as I said above, they might not (in fact, likely will not) **ever** be presented with an option to downgrade later on at all.  I believe it depends on their answers to the survey questions.

If it turns out people aren't noticing the current message, a solution could just be to make it more visible.  Maybe the "If you choose to downgrade, your plan will be downgraded immediately" message could be put in bold... or the title ("Would you rather switch to a more affordable plan?") could perhaps become "Would you rather switch to a more affordable plan, instead of cancelling?" to make it more clear that the user is completely leaving the cancellation flow.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3361887

Thank you @aaronrobertshaw for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3361887).

**Pull request merged**